ush/rocoto/setup_workflow.py

```diff
@@ -1159,21 +1154,21 @@ def get_hyb_tasks(dict_configs, cycledef='enkf'):
     if do_efsoi in ['Y', 'YES']:
     # esfcfsoi
          deps1 = []
-         data = '&ROTDIR;/%s.@Y@m@d/@H/%s.t@Hz.loganl.txt' % (cdump, cdump)
+         data = '&ROTDIR;/{cdump}.@Y@m@d/@H/{cdump}.t@Hz.loganl.txt' 
```

Comment:
This one is missing the f in front, so the variables won't be substituted

Reply:
Fixed. You have a better eye than me. Now I'm puzzled why the xml file worked...